src/pages/contact-us/SendGridForm/index.tsx

```diff
@@ -56,29 +60,48 @@ export default function (): React.ReactElement {
     );
     if (isExistError) return;
 
+    setSubminting(true);
+
     var formdata = new FormData();
     formdata.append("fullname", fullname.val);
     formdata.append("email", email.val);
     formdata.append("message", message.val);
     formdata.append("organization", organization.val);
     formdata.append("role", role.val);
 
-    const res = await fetch(LafyunFunction, {
+    const res = await fetch(CloudflareWorker, {
       body: formdata,
       method: "POST",
-    });
-
-    const { error } = await res.json();
-    if (error) {
-      alert(error);
-    } else {
-      alert("Email has successfully sent!");
-      setFullname(getResetField());
-      setEmail(getResetField());
-      setMessage(getResetField());
-      setOrganization(getResetField());
-      setRole(getResetField());
-    }
+    })
+      .then(() => {
+        alert("Email Has Successfully Sent!");
+        // setFullname(getResetField());
+        // setEmail(getResetField());
+        // setMessage(getResetField());
+        // setOrganization(getResetField());
+        // setRole(getResetField());
+      })
+      .catch((error) => {
+        console.error(error);
+        alert("Network Error.");
+      })
+      .finally(() => {
+        setSubminting(false);
+      });
+
+    // const { error } = await res.json();
+    // if (error) {
+    //   alert(error);
+    // } else {
+    //   alert("Email Has Successfully Sent!");
+    //   // setFullname(getResetField());
+    //   // setEmail(getResetField());
+    //   // setMessage(getResetField());
+    //   // setOrganization(getResetField());
+    //   // setRole(getResetField());
+    // }
```

Comment:
Remove useless  comments. 
if some code commented, then that means they can be cleared.